riotfile.py

```diff
@@ -936,6 +936,49 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            # aiobotocore: aiobotocore>=1.0 not yet supported
+            name="aiobotocore",
+            command="pytest {cmdargs} tests/contrib/aiobotocore",
+            venvs=[
+                # aiobotocore dropped Python 3.5 support in 0.12
+                Venv(
+                    pys=["3.5", "3.6"],
```

Comment:
the better way to organize this is probably to subdivide by `aiobotocore` versions and not Python versions:

e.g.

- aiobotocore >= 0.2, <0.12 -> select_pys(max_version='3.5')
- aiobotocore >= 0.12 -> select_pys(min_version='3.6')

I think if I understand the way this is broken down better?